.github/workflows/deprecated-branches.yml

```diff
@@ -0,0 +1,20 @@
+name: Merge protection against deprecated branches
+
+on:
+  pull_request:
+    # Every-time a branch is deprecated it should be added here
+    branches:
+      - '1.0'
+      - '1.1'
+      - '1.2'
```

Comment:
Can we add 2.0, 2.1, 2.2?

Reply:
Also 1.x

Reply:
What about 2.3? Is this going to be a new step on every release to append to this list?